### PR TITLE
OCM-10730 | feat: Day1 additional SG support for HCP

### PR DESCRIPTION
**Pull request by hunterkepley**
https://issues.redhat.com/browse/OCM-10730

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,4 +35,8 @@ override.tf.json
 
 # Ignore CLI configuration files
 .terraformrc
-terraform.rc
\ No newline at end of file
+terraform.rc
+
+# Ignore IDE directories
+.idea
+.idea/*
```
Comment (robpblake) [inline .gitignore:42]: @hunterkepley I think this can just be `.idea/` to ignore the directory as-per https://git-scm.com/docs/gitignore

**Comment by gdbranco** [comment]
@hunterkepley missing to run `make terraform-docs`. Also we need to wait for 1.6.6 release to support the security groups in provider

**Comment by hunterkepley** [comment]
```
+ echo 'It seems like you need to run '\''make terraform-docs'\''. Please run it and commit the changes'
It seems like you need to run 'make terraform-docs'. Please run it and commit the changes
```

Ran this and no changes 🤔

**Comment by gdbranco** [comment]
closing in favor of #54 , it seems that your version for terraform-docs is different or maybe due to using mac

**Pull request closed without merging**